pkg/execution/driver/driver.go

```diff
@@ -31,6 +31,7 @@ type FunctionStack struct {
 func MarshalV1(ctx context.Context, s state.State, step inngest.Step, stackIndex int, env string) ([]byte, error) {
 	data := map[string]interface{}{
 		"event": s.Event(),
+		// "events": s.Events(), disabling this for now so we don't send unnecessary data to customer infra
```

Comment:
per discussion in Discord, don't send list of events to SDK for now since it will reduce the quota for amount of data that can be sent by half.
the appropriate way to do it is to determine the fields to send over based on SDK version.

which I'll do as a follow up.